src/test/java/com/thealgorithms/dynamicprogramming/Sum_Of_SubsetTest.java

```diff
@@ -0,0 +1,21 @@
+package com.thealgorithms.dynamicprogramming;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class Sum_Of_SubsetTest {
```

Comment:
All of the classes names should be _PascalCase_. I would suggest to rename the `Sum_Of_Subset` class also. Please remember to change the corresponding filenames.